src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,93 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types";
+import { HomeAssistant } from "../../../types";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement implements LovelaceCard {
+  protected config?: Config;
+  private _hass?: HomeAssistant;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  set hass(hass: HomeAssistant) {
+    this._hass = hass;
+    for (const el of this.shadowRoot!.querySelectorAll("#root > *")) {
```

Comment:
I realized by reading #1792  that `hass` can be set before we have rendered for the first time, so we should actually check `if (this.shadowRoot)`

Reply:
Actually, no, nevermind. shadowRoot is attached in the constructor. We might just not have rendered yet, which is fine because it means we just find nothing